app/controllers/muted/users_controller.rb

```diff
@@ -12,7 +12,8 @@ class UsersController < ApplicationController
     # GET /users/:user_id/muted/users
     def index
       @mutes = @user.mutes_as_muter
-        .joins(:muted).includes(muted: :default_pseud)
+        .joins(:muted)
+        .includes(muted: { default_pseud: { icon_attachment: :blob }, pseuds: { icon_attachment: :blob } })
```

Comment:
To be the same as the blocked users:
```suggestion
        .includes(muted: [:pseuds, default_pseud: { icon_attachment: :blob }])
```